src/config.rs

```diff
@@ -426,9 +426,111 @@ pub struct DbOptions {
     ///
     /// Default: no TTL (insertions will remain until deleted)
     pub default_ttl: Option<u64>,
+
+    pub max_unflushed_memtable: usize,
 }
 
 impl DbOptions {
+    /// Converts the DbOptions to a JSON string representation
+    pub fn to_json_string(&self) -> Result<String, serde_json::Error> {
+        serde_json::to_string_pretty(self)
+    }
+
+    /// Logs the current configuration and enabled features.
+    pub fn log_config(&self) {
```

Comment:
I think we don't need this method now that we're logging it as json right?

Reply:
sounds good, lemme take a look and update.